tests/integration/test_ogdc_viz_recipe.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import pytest
+
+from ogdc_runner.api import submit_ogdc_recipe
+from ogdc_runner.argo import ARGO_WORKFLOW_SERVICE
+from ogdc_runner.exceptions import OgdcDataAlreadyPublished
+
+
+def test_submit_ogdc_viz_recipe_fast(test_viz_workflow_fast_recipe_directory):
+    """Test visualization workflow with a small geopackage for faster feedback."""
+    workflow_name = submit_ogdc_recipe(
+        recipe_dir=test_viz_workflow_fast_recipe_directory,
+        overwrite=True,
+        wait=True,
+    )
+
+    # Cleanup test workflow.
+    ARGO_WORKFLOW_SERVICE.delete_workflow(workflow_name)
+
+
+@pytest.mark.slow
+def test_submit_ogdc_viz_recipe(test_viz_workflow_recipe_directory):
```

Comment:
What's the main difference between this and the fast variant? Could we just eliminate the slow test, or does the slower one exercise some code that the other doesn't? It may be worth adding this detail to the docstring of the test and/or fixture to explain.

Reply:
I would say we remove the slower one/update it to use the fast config and just have one test recipe for viz workflow unless @rushirajnenuji says otherwise!